ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_QMLSamples/mainSample.cpp

```diff
@@ -0,0 +1,680 @@
+  StyleWmsLayer::init();
+  SetSurfacePlacementMode::init();
+  SymbolizeShapefile::init();
+  Symbols::init();
+  SyncMapViewSceneView::init();
+  TakeScreenshot::init();
+  TerrainExaggeration::init();
+  TileCacheLayer::init();
+  TokenAuthentication::init();
+  TraceUtilityNetwork::init();
+  Unique_Value_Renderer::init();
+  UpdateAttributesFeatureService::init();
+  UpdateGeometryFeatureService::init();
+  ValidateUtilityNetworkTopology::init();
+  VectorTiledLayerUrl::init();
+  ViewContentBeneathTerrainSurface::init();
+  ViewPointCloudDataOffline::init();
+  ViewshedCamera::init();
+  ViewshedGeoElement::init();
+  ViewshedLocation::init();
+  Web_Tiled_Layer::init();
+  WmsLayerUrl::init();
+  WMTS_Layer::init();
+
+#ifdef SHOW_PORTAL_SAMPLES
+  PortalUserInfo::init();
+  SearchForWebmap::init();
+  AddItemsToPortal::init();
+  ShowOrgBasemaps::init();
+#endif // HIDE_PORTAL_SAMPLES
+
+#ifdef SHOW_RASTER_FUNCTION_SAMPLE
+  RasterFunctionFile::init();
+#endif // SHOW_RASTER_FUNCTION_SAMPLE
+
+#ifdef LOCALSERVER_SUPPORTED
+  LocalServerServices::init();
+  LocalServerMapImageLayer::init();
+  LocalServerFeatureLayer::init();
+  LocalServerGeoprocessing::init();
+#endif // LOCALSERVER_SUPPORTED
+
+#endif // CPP_VIEWER
+}
+
+void registerClasses()
+{
+  // qmlRegisterSingletonType<SampleManager>("Esri.ArcGISRuntimeSamples", 1, 0, "SampleManager",
+  //                                                 &esriSampleManagerProvider);
+
+  qmlRegisterSingletonType<SyntaxHighlighter>("Esri.ArcGISRuntimeSamples", 1, 0, "SyntaxHighlighter",
+                                              &syntaxHighlighterProvider);
+  // qmlRegisterUncreatableType<DataItem>("Esri.ArcGISRuntimeSamples", 1, 0,
+  //                                      "DataItem", "DataItem is an uncreatable type");
+  // qmlRegisterUncreatableType<DataItemListModel>("Esri.ArcGISRuntimeSamples", 1, 0,
+  //                                               "DataItemListModel", "DataItemListModel is an uncreatable type");
+  // qmlRegisterUncreatableType<CategoryListModel>("Esri.ArcGISRuntimeSamples", 1, 0,
+  //                                               "CategoryListModel", "CategoryListModel is an uncreatable type");
+  // qmlRegisterUncreatableType<SampleListModel>("Esri.ArcGISRuntimeSamples", 1, 0,
+  //                                             "SampleListModel", "SampleListModel is an uncreatable type");
+  // qmlRegisterUncreatableType<SampleCategory>("Esri.ArcGISRuntimeSamples", 1, 0,
+  //                                            "SampleCategory", "SampleCategory is an uncreatable type");
+  // qmlRegisterUncreatableType<Sample>("Esri.ArcGISRuntimeSamples", 1, 0,
+                                     // "Sample", "Sample is an uncreatable type");
+  // qmlRegisterUncreatableType<SourceCodeListModel>("Esri.ArcGISRuntimeSamples", 1, 0,
+  //                                                 "SourceCodeListModel", "SourceCodeListModel is an uncreatable type");
```

Comment:
Take a look. If these aren't needed, just remove them. If there are other things that need to be cleaned up in here, please do so. I didn't look closely at all the files.